templates/server/index.html

```diff
@@ -39,6 +39,29 @@ <h1>Scale out with Ubuntu Server</h1>
   </div>
 </div>
 
+<div class="p-strip is-bordered">
+  <div class="u-fixed-width">
+    <h2>
+      What&rsquo;s new in {{ releases.latest }} LTS
```

Comment:
```suggestion
      What&rsquo;s new in {{ releases.latest.full_version }} LTS
```
you are outputting the entire yaml string

Reply:
Is there a reason why we are writing full version here and short version in the previous one?

Reply:
there is _almost_ never a dot release of a latest, if there was, it wouldn't change the features, where an LTS might change more.